src/tig.c

```diff
@@ -266,7 +266,11 @@ view_driver(struct view *view, enum request request)
 		break;
 
 	case REQ_PARENT:
-		report("Moving to parent is not supported by the %s view", view->name);
+		if (! strcmp(view->name, "main")) {
+			move_view(view, request);
```

Comment:
The changes to support `REQ_PARENT` should go in `src/main.c` which is all the main view specific code. There's a `main_request` method where the `case REQ_PARENT` and related `move_view` changes in this patch can exist.